Firebase Push Notifications.md

```diff
@@ -0,0 +1,88 @@
+# Firebase push notifications
+
+Table of contents:
+
+1. [General](#general)
+2. [Setup](#setup)
+3. [Development approaches](#development-approaches)
+4. [Choosing right development approach](#choosing-development-approach)
```

Comment:
I would leave out this table of content, our other chapters don't have it and the 4th bullet already doesn't work :)